tests/util/setup-smoke-tests.ts

```diff
@@ -36,17 +40,19 @@ export function describeSmokeSuite(
     before("Starting Moonbeam Smoke Suite", async function () {
       this.timeout(10000);
 
-      [context.polkadotApi, context.relayApi] = await Promise.all([
+      [context.polkadotApi, context.relayApi, context.ethers] = await Promise.all([
         ApiPromise.create({
           initWasm: false,
           provider: new WsProvider(options.wssUrl),
+          typesBundle: types,
         }),
         options.relayWssUrl
           ? ApiPromise.create({
               initWasm: false,
               provider: new WsProvider(options.relayWssUrl),
             })
           : unimplementedApi(),
+        new ethers.providers.JsonRpcProvider(options.ethRpcUrl),
```

Comment:
This needs to be cleaned up at the end, otherwise NodeJS will not exit

Reply:
As far as I can tell, there isn't a `close()` or `disconnect()` method for this provider library given that it's opening an asynchronous push connection, unlike apipromise's websocket.